File: Day_18.java
public class Solution {

    private LinkedList<Character> stack;
    private LinkedList<Character> queue;

    public Solution() {
        stack = new LinkedList<>();
        queue = new LinkedList<>();
    }

    public void pushCharacter(char ch) {
        stack.add(Character.valueOf(ch));
    }

    public void enqueueCharacter(char ch) {
        queue.add(Character.valueOf(ch));
    }

    public char popCharacter() {
        return stack.removeLast().charValue();
    }

    public char dequeueCharacter() {
        return queue.remove(0).charValue();
    }

File: Day_18.py
class Solution:
    stack = []
    queue = []
    
    def __init(self):
        self.stack = []
        self.queue = []
    
    def pushCharacter(self, ch):
        self.stack.append(ch)
        
    def enqueueCharacter(self, ch):
        self.queue.append(ch)
        
    def popCharacter(self):
        return self.stack.pop()
        
    def dequeueCharacter(self):
        return self.queue.pop(0)
        
